app/models/concerns/webhooks/outgoing/issuing_model.rb

```diff
@@ -49,7 +49,7 @@ def generate_webhook_perform(action)
     # Does Jbuilder support generating a hash instead of a JSON string?
     data = JSON.parse(to_api_json)
     webhook = send(BulletTrain::OutgoingWebhooks.parent_association).webhooks_outgoing_events.create(event_type_id: event_type.id, subject: self, data: data)
-    webhook.deliver
+    webhook.deliver(self)
```

Comment:
Do we need to even pass `self` here? I think `deliver` can also reference `self`, and `endpoints` method can too.